src/ai/backend/manager/cli/image_impl.py

```diff
@@ -192,3 +194,58 @@ async def dealias(cli_ctx, alias):
             log.exception("Alias not found.")
             return
         await session.delete(alias_row)
+
+
+async def validate_image_alias(cli_ctx, alias: str) -> None:
+    async with (
+        connect_database(cli_ctx.local_config) as db,
+        db.begin_session() as session,
```

Comment:
```suggestion
        db.begin_readonly_session() as session,
```
Since the DB session is used for `SELECT` only, let's start readonly session here.